specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2023-03-01-preview/Capabilities.json

```diff
@@ -0,0 +1,562 @@
+          "x-ms-identifiers": [
+            "name"
+          ],
+          "description": "The list of server versions supported for this capability.",
+          "readOnly": true
+        },
+        "fastProvisioningSupported": {
+          "type": "string",
+          "description": "Gets a value indicating whether fast provisioning is supported",
+          "readOnly": true,
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "fastProvisioningSupportedEnum",
+            "modelAsString": true
+          }
+        },
+        "supportedFastProvisioningEditions": {
+          "type": "array",
+          "description": "List of supported server editions for fast provisioning",
+          "items": {
+            "$ref": "#/definitions/FastProvisioningEditionCapability"
+          },
+          "x-ms-identifiers": [
+            "supportedSku"
+          ],
+          "readOnly": true
+        },
+        "geoBackupSupported": {
+          "type": "string",
+          "description": "Determines if geo-backup is supported in this region.",
+          "readOnly": true,
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "geoBackupSupportedEnum",
+            "modelAsString": true
+          }
+        },
+        "zoneRedundantHaSupported": {
+          "type": "string",
+          "description": "A value indicating whether Zone Redundant HA is supported in this region.",
+          "readOnly": true,
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "zoneRedundantHaSupportedEnum",
+            "modelAsString": true
+          }
+        },
+        "zoneRedundantHaAndGeoBackupSupported": {
+          "type": "string",
+          "description": "A value indicating whether Zone Redundant HA and Geo-backup is supported in this region.",
+          "readOnly": true,
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "zoneRedundantHaAndGeoBackupSupportedEnum",
+            "modelAsString": true
+          }
+        },
+        "storageAutoGrowthSupported": {
+          "type": "string",
+          "description": "A value indicating whether storage auto-grow is supported in this region.",
+          "readOnly": true,
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "storageAutoGrowthSupportedEnum",
+            "modelAsString": true
+          }
+        },
+        "onlineResizeSupported": {
+          "type": "string",
+          "description": "A value indicating whether online resize is supported in this region for the given subscription.",
+          "readOnly": true,
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
```

Comment:
Why not call the values 'Supported' and 'Unsupported' then?

Reply:
To be consistent with other properties and this is feature at regional level and it is Boolean in the API side. Initially all of them are Boolean and we got comments to change it to enable/disable. so, we made these changes,

Reply:
Updated description